Pipfile

```diff
@@ -0,0 +1,14 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+pytest = "~=4.6.3"
+flake8 = "~=3.7.7"
+mypy = "==0.720"
+
+[requires]
+python_version = "3.8"
```

Comment:
Can you remove the Pipfile? We had trouble in the past with pipenv and I'd like to avoid the duplication with the requirements.txt. Also if we'd switch again I'd prefer it in a separate pull request.